podvm/qcow2/centos/qemu-centos.pkr.hcl

```diff
@@ -69,6 +69,7 @@ build {
   provisioner "shell" {
     remote_folder = "~"
     inline = [
+      "sudo dnf -y install open-vm-tools || exit 0",
```

Comment:
how about adding this to misc_settings.sh file ?
Also any thoughts on how this can be installed only for vsphere ?

Reply:
I used the script because misc_settings.sh is common for all including ubuntu. I think we should follow the ubuntu example and install open-vm-tools by default. That way, we don't have to worry about CLOUD_PROVIDER specific qcow2 images. What do you think ?

Reply:
Got it. Can you also add it to ubuntu ?
Also later on it will be good to figure out a way to install specific packages for specific providers

Reply:
Actually, ubuntu already has open-vm-tools installed in the upstream cloud image. So, when converting the ubuntu qcow2, we don't need to install additional packages.

Reply:
Thanks for clarifying @whitebrandy